public/index.html

```diff
@@ -36,48 +38,49 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script>
-      var schema = {
-        'GitHub issue': {
-          'type': 'string'
-        },
-        'Signs and Symptoms': {
-          'type': 'string'
-        },
-        'Treatments': {
-          'type': 'string'
-        },
-        'Fix Difficulty': {
-          'type': 'number'
-        },
-        'Pull Leech Logs': {
-          'type': 'boolean'
-        },
-        'Last Updated': {
-          'type': 'date'
-        },
-        'Versions Affected': {
-          'type': 'semver-range'
-        },
-        'Fixed in Version': {
-          'type': 'semver'
+  <script>
+    var schema = {
+      'GitHub issue': {
+        'type': 'string'
+      },
+      'Signs and Symptoms': {
+        'type': 'string'
+      },
+      'Treatments': {
+        'type': 'string'
+      },
+      'Fix Difficulty': {
+        'type': 'number'
+      },
+      'Pull Leech Logs': {
+        'type': 'boolean'
+      },
+      'Last Updated': {
+        'type': 'date'
+      },
+      'Versions Affected': {
+        'type': 'semver-range'
+      },
+      'Fixed in Version': {
+        'type': 'semver'
+      }
+    };
+
+    document.addEventListener("DOMContentLoaded", function () {
+      Pensieve({
```

Comment:
The initial use case is that we can just load it into a html page on any repo with no other configuration.